README.md

```diff
@@ -54,7 +54,7 @@
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Live Demo Link](https://seyioyemade.github.io)
+- [Live Demo Link](https://seyioyemade.github.io/portfolio/)
 
 
```

Comment:
- [ ] Kindly ensure that all your authors contributors are present in the `Authors` section of the README to make your documentation more professional